server/views/components/key-info-items/template.njk

```diff
@@ -0,0 +1,37 @@
+<div class="govuk-grid-column-one-quarter" id="keyInfo">
```

Comment:
I might have got this wrong 🤔 
Because the CSS uses the class `key-info-item` not the tags. My bad.